components/gantry/multiaxis/multiaxis.go

```diff
@@ -115,9 +126,16 @@ func (g *multiAxis) MoveToPosition(ctx context.Context, positions []float64, ext
 		}
 
 		pos := positions[idx : idx+len(subAxNum)]
+		var speed []float64
```

Comment:
I was thinking the speeds from multi-axis should pretty much just be passed to the single-axis as is and be handled by the single axis since the default could be specific to each axis? thoughts on that? the only issue is basically when it's an empty list because then I will get out of range errors, which is why I have lines 131-133 but I can change this to just be `[]float64{}` instead of `[]float64{-1}`?

Reply:
I just figured since we were already iterating through subaxes we could let them each individually handle the different speed cases because that logic would still need to be present in a the single-axis code anyways

Reply:
so, I prefer that if we pass in values that are meant to say: this is undefined, as floats, we pass through golang's definition of an undefined float `math.NaN`.

I'm not convinced that a negative speed will always be a no-go for the single-axis code. We should at least comment in that code that we're expecting positive speeds and add a debug statement to that effect.

Reply:
yeah makes sense. I changed it to `[]float64{}` instead of `[]float64{-1}`